README.md

```diff
@@ -1,53 +1,121 @@
+
+## [Hedera Projects](#hedera-projects)
+Featuring projects built on top of the [Hedera Hashgraph](https://www.hedera.com) network. 
+
+**Advertising**
+
+* [AdsDax](https://www.hedera.com/users/adsdax) - AdsDax is a blockchain advertising platform and marketplace to create, buy, verify and sell advertising.
+
+**Decentralized Identity**
+
+* [MyEarth.id](https://www.hedera.com/users/earth-id) - Earth ID is a decentralized identity management platform, which aims to provide a secure, consolidated identity.
+
+**Explorers**
+
+* [Hash-Hash.info](https://hash-hash.info) - The first explorer of Hedera Hashgraph.
+* [HashArc](https://www.hasharc.com/) - A network explorer provided by the [OpenPharma](https://www.openpharma.io/) team.
+* [DragonGlass](https://app.dragonglass.me/hedera/home) - Live and historical data across distributed ledgers.
+
+**Games**
+
+* [Playhash](https://www.playhash.com/) - PlayHash.Com is a fair online lottery platform created by Hashgraph enthusiasts.
+
+**Network Data APIs**
+
+* [DragonGlass](https://app.dragonglass.me/hedera/apiview) - Providing developers with REST-based APIs for all Transactions, Accounts, and Smart Contract Information generated in Hedera Hashgraph.
+* [Kabuto](https://docs.kabuto.sh/) - Developer-centric API for accessing Hedera Hashgraph data such as account, transaction, and balance information. Currently in beta.
+
+**Pharmaceuticals** 
+
+* [OpenPharma](https://www.hedera.com/users/certara) - An industry-first Open API software platform for pharma and life sciences, built by [Certara](https://www.certara.com/).
+
+**Supply Chain**
+
+* [Armada Chain](https://www.hedera.com/users/armada-chain) - Armada is a collaboration hub that authenticates transactions, streamlines communication, and connects supply chains without compromising existing workflows.
+
+**Wallets**
+
+* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera iOS wallet](https://itunes.apple.com/us/app/hedera-wallet/id1442631626?mt=8) - The open source Hedera Wallet for iOS lets you manage HBAR and create accounts on the Hedera mainnet.
+* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera Android Wallet](https://play.google.com/store/apps/details?id=com.hedera.wallet&hl=en_US) - The open source Hedera Wallet for Android lets you manage HBAR and create accounts on the Hedera mainnet.
+* [MyHbarWallet.com](https://myhbarwallet.com/) - A hosted implementation of the MyHbarWallet open source project.
+* [Composer For Hedera Hashgraph](https://chrome.google.com/webstore/detail/composer-for-hedera-hashg/hdjnnemgikeoehneddegfcmkljenlean) - A community maintained tool to interact with Hedera from the Chrome environment.
+* [Zabo](https://zabo.com/) - Providing application developers a way to connect any cryptocurrency and any wallet to any application.
```

Comment:
Should this go into a developer tools section? It's not really a wallet in itself.

Reply:
Which of these would you not consider a wallet? I'd think 

- android = mobile wallet
- myhbarwallet = browser wallet (client side)
- composer = browser wallet (extension)
- Zabo = embeddable wallet

We can include them in both if you think that makes more sense, but I think that having a section of "reference wallets" seems helpful?